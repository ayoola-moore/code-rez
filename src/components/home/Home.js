import React from 'react'
import './home.scss'

const Home = () => {
    return (
        <div className={'home_wrapper'}>
home
        </div>
    )
}

export default Home
