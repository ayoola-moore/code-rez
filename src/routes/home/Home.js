import * as React from 'react' // Flow recommended this
import { withRouter } from 'react-router-dom'
import Button from '../../components/buttons'
import './home.scss'

type HomeProps = {
    history: Object,
}

class Home extends React.Component<HomeProps> {
    handleClick = () => this.props.history.push('/recommendations')
    render() {
        return (
            <div className={'home_wrapper'}>
                <Button handleClick={this.handleClick} className={'lunch'}>
                    What's for lunch
                </Button>
            </div>
        )
    }
}

export default withRouter(Home)
