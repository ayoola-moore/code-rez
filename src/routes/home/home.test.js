import * as React from 'react' // Flow recommended this
import { shallow } from 'enzyme'
import Home from './Home'

describe('Home', () => {
    it('should render Home Component as the landing recommendations', () => {
        const component = shallow(<Home />)
        expect(component).toMatchSnapshot()
    })
})
