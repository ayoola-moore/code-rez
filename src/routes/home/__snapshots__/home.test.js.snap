// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home Component as the landing recommendations 1`] = `
<Route>
  <Component />
</Route>
`;

exports[`Home should render Home Component as the landing view 1`] = `
<Route>
  <Component />
</Route>
`;
